catanatron_gym/catanatron_gym/board_tensor_features.py

```diff
@@ -67,7 +67,6 @@ def get_tile_coordinate_map():
 
 # Create mapping of node_id => i,j and edge => i,j. Respecting (WIDTH, HEIGHT)
 def init_board_tensor_map():
-    global STATIC_GRAPH
```

Comment:
I removed these because I don't believe they're needed. IIUC `global` is only needed when referring the variable might make a new local version.
```
a = 1
print(f"global {a=}")
def foobar1():
    a = 2
    print(f"local {a=}")

foobar1()
print(f"global {a=}")
```
Results in 
```
$ global a=1
$ local a=2
$ global a=1
```

 Using global now:
```
a = 1
print(f"global {a=}")
def foobar2():
    global a
    a = 2
    print(f"local {a=}")

foobar2()
print(f"global {a=}")
```
Results in 
```
$ global a=1
$ local a=2
$ global a=2
```

Without global, however, I think referencing the global variable works normally. Most of the time, the code is not changing STATIC_GRAPH with a statement like:

```
STATIC_GRAPH = <something else>
```

It's instead operating on the existing STATIC_GRAPH. I could separate this into a separate PR to trim this one down.

Reply:
agreed on everything - we can split cleanup like this into a pre-PR.